ui/apps/platform/cypress/integration/dashboard.test.js

```diff
@@ -61,6 +61,24 @@ describe('Dashboard page', () => {
         cy.get(selectors.navLink).should('have.class', 'pf-m-current');
     });
 
+    it('should render navbar with Dashboard selected', () => {
```

Comment:
@dvail 

Calling out where Mark is moving these tests, because the move will help you with the new PF dashboard.